### PR TITLE
README rework: link into language/dart page on grpc.io etc

### PR DESCRIPTION
**Pull request by chalin**
Contributes to https://github.com/grpc/grpc.io/issues/180

cc @sigurdm @mit-mit @ejona86

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,27 +1,38 @@
-The [Dart](https://www.dart.dev/) implementation of
-[gRPC](https://grpc.io/): A high performance, open source, general RPC framework that puts mobile and HTTP/2 first.
+# gRPC Dart
```
Comment (kevmoo) [inline README.md:1]: Remove title – this is redundant to the title on the GitHub page and package page

Comment (chalin) [reply README.md:1]: Many (if not most) pages on GH use a first-level heading as a title, e.g.:

- https://pub.dev/packages/analyzer -> [README](https://github.com/dart-lang/sdk/blob/master/pkg/analyzer/README.md)
- https://pub.dev/packages/crypto -> [README](https://github.com/dart-lang/crypto/blob/master/README.md)
- https://pub.dev/packages/meta -> [README](https://github.com/dart-lang/sdk/blob/master/pkg/meta/README.md)
- https://github.com/dart-lang/protobuf -> [README](https://github.com/dart-lang/protobuf/blob/master/README.md)

To list a few, which are Dart packages. And all grpc-* repos use a title.

Comment (kevmoo) [reply README.md:1]: Right. But imagine how this will look at https://pub.dev/packages/grpc

Double titles!

Comment (chalin) [reply README.md:1]: But those titles serve difference purposes IMHO.

Ok, your call.

Comment (chalin) [reply README.md]: Done

**Comment by mit-mit** [comment]
Thanks @chalin for the PR! We prefer using in-place links, rather than putting them all at the end (as these are hard to keep in sync when the text is edited).

**Comment by chalin** [comment]
> ... We prefer using in-place links, rather than putting them all at the end (as these are hard to keep in sync when the text is edited).

@mit-mit: All links have been inlined, per your feedback. /cc @kwalrath

**Comment by chalin** [comment]
Anything left to do before this is merged?

**Pull request merged**